omnibus/resources/agent/msi/localbuild/parameters.wxi

```diff
@@ -8,4 +8,6 @@
   <?define BinFiles="c:\omnibus-ruby\src\datadog-agent\src\github.com\DataDog\datadog-agent\bin\agent" ?>
   <?define EtcFiles="C:/omnibus-ruby/src\etc\datadog-agent" ?>
   <?define ResourcesDir="c:\dev\go\src\github.com\datadog\datadog-agent\omnibus\resources\agent\msi"?>
+  <?define IncludePython2= "true" ?>
+  <?define IncludePython3= "false" ?>
```

Comment:
why are these 2 lines included in the PR?

Reply:
For anyone who wants to be able to do a test MSI without going through a whole `omnibus` build.  The whole `localbuild` directory exists to be able to do an MSI  test->change->build cycle without waiting for an omnibus build.